pkg/arvo/gen/quiz.hoon

```diff
@@ -0,0 +1,7 @@
+/+  quiz
+:-  %say
+|=  [[now=@da eny=@uvJ bec=beak] [[vax=vase ~] ~]]
+:: Default to using 0 entropy and no special quizzers.
+=+  qui=~(. quiz [eny 100])
+=+  che=^-(? (check:qui vax))
+[%noun che]
```

Comment:
Nit but since you're not doing anything with the intermediate values i'd personally prefer fewer names to keep track of.

```suggestion
[%noun (~(check quiz eny 100) vax)]
```